src/annotationLayer.js

```diff
@@ -757,8 +758,9 @@ var annotationLayer = function (arg) {
           m_this.map().interactor().addAction(action);
         });
       }
+      console.log({mode: m_mode, oldMode, oldState, oldCoordinates, reason});
```

Comment:
@manthey Was this meant to stay in?

Reply:
Nope.  And pre-commit and CI both should have refused it...